### PR TITLE
Update groups according to FESCo exception requests

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: data/groups.yaml [hunk 1]
```diff
@@ -11,18 +11,51 @@ gimp:
...
     - pycairo
 olpc:
-    name: OLPC/Sugar
+    name: Sugar desktop environment
+    exception: https://pagure.io/fesco/issue/2267
     hidden: true
     packages:
-    # dnf repoquery --repo rawhide 'sugar-*' 'olpc-*' --qf '- %{name}'
     - olpc-kbdshim
-    - olpc-netutils
```
Comment (hroncok) [inline data/groups.yaml]: I think this should remain here.It is a Python 2 OLPC package, it just has x86_64 excluded, so it sometimes doesn't show up.

File: data/groups.yaml [hunk 2]
```diff
@@ -176,16 +206,35 @@ mozjs60:
...
     - gr-fcdproplus
     - gr-hpsdr
-    - gr-iio
     - gr-iqbal
     - gr-osmosdr
     - gr-rds
+    - python-zmq
+
+    # gr-iio, gnuradio + its immediate dependencies + their runtime deps
+    # (These aren't listed in the exception request at time of writing, but
```
Comment (hroncok) [inline data/groups.yaml]: neither is python-zmq

File: data/groups.yaml [hunk 3]
```diff
@@ -176,16 +206,35 @@ mozjs60:
...
+    - python-lxml
+    - python-mako
+    - python-markupsafe
+    - python-pyopengl
+    - python2-numpy
+    - python-six
+    - pygtk2
+    - pycairo
+    - pygobject2
+    - wxPython
```
Comment (hroncok) [inline data/groups.yaml]: this is now retired

File: data/groups.yaml [hunk 4]
```diff
@@ -176,16 +206,35 @@ mozjs60:
     - mozjs60
 gnuradio:
     name: GNU Radio
+    exception: https://pagure.io/fesco/issue/2272
```
Comment (hroncok) [inline data/groups.yaml]: Closed, all ported. Give @yarda a medal.

Comment (encukou) [reply data/groups.yaml]: Wow! What great news!

This comment appeared to me after I pushed a commit to fixup the group. I'll amend it.

Comment (encukou) [reply data/groups.yaml]: I'll remove the group when it disappears in the compose.

File: data/groups.yaml [hunk 5]
```diff
@@ -11,18 +11,52 @@ gimp:
...
+    - pybox2d
+    - python-BeautifulSoup
+    - gnome-python2
+    - gnome-python2-desktop
+    - pygtk2
+    - python-docutils
+    - python-dateutil
+    - python-six
+    - python-simplejson
+    - python-nose
```
Comment (hroncok) [inline data/groups.yaml]:
```suggestion
```

**Pull request merged**